src/com/google/enterprise/adaptor/filenet/ConfigOptions.java

```diff
@@ -144,6 +146,24 @@ public ConfigOptions(AdaptorContext context)
         splitter.split(config.getValue("filenet.includedMetadata")));
     logger.log(Level.CONFIG, "filenet.includedMetadata: {0}", includedMetadata);
 
+    final String metadataDateFormat =
+        config.getValue("filenet.metadataDateFormat");
+    this.metadataDateFormat =
+        new ThreadLocal<SimpleDateFormat>() {
+          @Override protected SimpleDateFormat initialValue() {
+            return new SimpleDateFormat(metadataDateFormat);
+          }
+        };
+    try {
+      this.metadataDateFormat.get();
+      this.metadataDateFormat.remove();
```

Comment:
This thread isn't going to need a date formatter. I would like to call initialValue, but I can't do that without using a member class. I could also just copy the new SimpleDateFormat(metadataDateFormat) expression rather than reuse the one in the ThreadLocal.